src/services/UserService.ts

```diff
@@ -0,0 +1,50 @@
+const dotEnv = require('dotenv');
+import jwt from 'jsonwebtoken';
+import BaseService from "../base/BaseService";
+
+dotEnv.config();
+
+const { User } = require('../db/models');
+
+export default class UserService extends BaseService<any> {
+  public username;
+  public email;
+
+  constructor() {
+    super(User);
+  }
+
+  static async findAllUsers() {
+    const results = await User.findAll({})
+    return results
+  }
+
+  async findUserByUsername(username: string) {
+    try {
+      const response = await this.findOne({where: username});
+      if (!response) return false
+      return response
+    } catch (error) {
+      return false
+    }
+  }
+
+  async findUserByEmail(email: string) {
```

Comment:
THis is not necessary since the findOne method can take in options have this a bit redundant 
i.e `this.service.findOne({where: {email})`